docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -1,7 +1,7 @@
 = Install OpenShift 4 on cloudscale.ch
-:ocp-minor-version: 4.16
-:k8s-minor-version: 1.29
-:ocp-patch-version: {ocp-minor-version}.21
+:ocp-minor-version: 4.17
+:k8s-minor-version: 1.30
+:ocp-patch-version: {ocp-minor-version}.3
```

Comment:
I don't see a 4.17.3 base image folder in https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.17/, are you sure that's correct? I put `.2` for Exoscale

```suggestion
:ocp-patch-version: {ocp-minor-version}.2
```

Reply:
Ah right my installer is .3, i think the image i used was .0